app/public/templates/home.html

```diff
@@ -48,9 +48,13 @@
           alt=""
         />
       </nav>
-      {% if len(res_scenarios) > 0 %}
-      <h1>Try out one of the escape rooms below!</h1>
-      {% for scenario in res_scenarios %}
+      {% if len(res_scenarios) == 0 %}
+      <h2>No escape rooms found</h2>
+      {% else %} {% if len(res_scenarios) > 1 %}
```

Comment:
This doesn't quite have the effect you want. In tornado templates, the else if statements are `elif`. So you want something along the lines of:
```
{% if *condition* %}...{% elif *condition* %}...{% else %}...{% end %}
```

Reply:
Did not know that! Thank you. I tried google for the else if condition but people were just doing nested if's 😅